ember-bootstrap/addon/components/bs-modal/dialog.ts

```diff
@@ -52,9 +84,9 @@ export default class ModalDialog extends Component {
    * @private
    */
   @action
-  getOrSetTitleId(modalNode) {
-    //Title element may be set by user so we have to try and find it to set the id
-    let nodeId = null;
+  getOrSetTitleId(modalNode: Element) {
+    //Title element may be set by user, so we have to try and find it to set the id
```

Comment:
While we fix the typo, let's also add the missing white space at the beginning.

```suggestion
    // Title element may be set by user, so we have to try and find it to set the id
```